cmd/installer/cli/root.go

```diff
@@ -68,6 +69,9 @@ func RootCmd(ctx context.Context, name string) *cobra.Command {
 				metrics.DisableMetrics()
 			}
 
+			// set the umask to 022 so that we can create files/directories with 755 permissions
+			_ = syscall.Umask(0o022)
```

Comment:
logrus.Debug the error

Reply:
The return value is not an error. Its the [previous umask](https://pkg.go.dev/syscall#Umask).